src/schemas/21P-527EZ/schema.js

```diff
@@ -264,19 +270,22 @@ const schema = {
             type: 'string',
           },
           childSocialSecurityNumber: schemaHelpers.getDefinition('ssn'),
+          /*
+            Where will the childRelationshipOptions live? Do we add the helper functions to this project?
```

Comment:
See comment on checkboxGroupSchema. Source of radioSchema in vets-website is https://github.com/department-of-veterans-affairs/vets-website/blob/39e38ef10d72d94b33d18e699a1495b1ae8bf68d/src/platform/forms-system/src/js/web-component-patterns/radioPattern.jsx#L79 (for reference)